app-creator/src/widgets/app-root.ts

```diff
@@ -0,0 +1,33 @@
+import {LitElement, html, customElement, css} from 'lit-element';
```

Comment:
Add a file level comment explaining what the component is for (and for other files): 

/**
 *  The root app component.
 */